.github/workflows/cmake.yml

```diff
@@ -1,8 +1,6 @@
 name: build-test
 on: 
   push:
-  pull_request:
```

Comment:
Assuming this runs the github actions on PRs, I think we should keep this.

Reply:
The reasons I removed it is because I realized it was redundant since the previous line already makes it run on every push including in pull requests. This line just makes the last test run twice.

Reply:
OK. Sounds good. Thank you. :+1: